### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Amir-61**
* Update readme

/to: @crandmck

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-## loopback-connector-sqlite3
+# loopback-connector-sqlite3
 
-The SQLite3 Connector module for for [loopback-datasource-juggler](http://docs.strongloop.com/loopback-datasource-juggler/).
+The SQLite3 Connector module for [loopback-datasource-juggler](https://github.com/strongloop/loopback-datasource-juggler).
```
Comment (0candy) [inline README.md]: I think the url should be `http://loopback.io/doc/en/lb2/SQLite3.html`

**Comment by Amir-61** [comment]
@slnode test please

File: README.md (hunk 1 above)
Comment (Amir-61) [inline README.md]: Based on slack conversation with @superkhau and @0candy I changed it to:
```
The official SQLite3 Connector module for the LoopBack framework.
```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-## loopback-connector-sqlite3
+# loopback-connector-sqlite3
 
-The SQLite3 Connector module for for [loopback-datasource-juggler](http://docs.strongloop.com/loopback-datasource-juggler/).
+The official SQLite3 Connector module for the LoopBack framework.
```
Comment (superkhau) [inline README.md]: lowercase c for Connector?

**Pull request merged**